cmd/chihaya/config.go

```diff
@@ -78,7 +78,11 @@ func (cfg ConfigFile) CreateHooks() (preHooks, postHooks []middleware.Hook, err
 			if err != nil {
 				return nil, nil, errors.New("invalid JWT middleware config: " + err.Error())
 			}
-			preHooks = append(preHooks, jwt.NewHook(jwtCfg))
+			hook, err := jwt.NewHook(jwtCfg)
+			if err != nil {
+				return nil, nil, errors.New("failed to start JWT middleware server: " + err.Error())
```

Comment:
I wouldn't assume anything about what failed during the attempt to construct the JWT middleware, I'd just say "failed to start JWT middleware" or something like that